api/descriptions/policies/compliance_ci_images_put.md

```diff
@@ -0,0 +1,49 @@
+Updates the compliance policy for images in your continuous integration (CI) pipeline.
+All rules in the policy are updated in a single shot.
+
+The policy set in this endpoint is enforced by the scanners in the Jenkins plugin and the `twistcli` command line tool.
+
+This endpoint maps to the policy table in **Defend > Compliance > Containers and images > CI** in the Console UI.
+
+
+### cURL Request
+
+The following cURL command overwrites all rules in your current policy with a new policy that has a single rule.
+
+```bash
+$ curl 'https://<CONSOLE>/api/v1/policies/compliance/ci/images?project=<PROJECT_NAME>' \
```

Comment:
@KICKdesigns For compliance rules, we need to also specify which "checks" should be validated by the scanner. Without specifying one or more checks, the rule is essentially useless. In a nutshell, each check validates some security best practice or setting. 

If you create a compliance rule in the UI, you'll see what I mean. The table under "Compliance actions" shows all the checks prebuilt into the product.

Please play around with the API, and figure out the right request to make. For this example, let's specify check `41`, which would be very relevant to this example.

<img width="1292" alt="compliance-api" src="https://user-images.githubusercontent.com/6518946/105149363-14debc80-5ac9-11eb-83ea-9d79b358111c.png">

Reply:
@iansk I can add that into the PUT doc but it's not required when submitting the PUT via curl. I'll also include the note about recommending using a check.

Reply:
See changes - let me know if you want additional edits based on the comment above.

Reply:
Thanks @KICKdesigns. Let's follow the "recipe" I outlined for developing curl commands. It's going to be impossible for you to guess what we consider "minimum" over and above what's technically minimum. So shoot for technically minimum, and I can help fill you in on the gaps during the review cycles.

Reply:
@iansk Got it. My only concern is that generally it seems the minimum is just the name and policyType. Do you want to include the thresholds and one of the conditions as a general practice - at least for the compliance endpoints? Perhaps we should take this approach on a section-by-section basis. (IE vulnerabilities need to keep the block and alert thresholds, compliance needs the alert and one condition, etc)

Reply:
@KICKdesigns Yes, totally agree. We're going to need to take this on a section-by-section basis. 

For the compliance, we've both now worked through the first endpoint, and come up with a consensus "minimum", which is:

* name
* policyType
* collection
* check + effect (alert or block)

Reply:
@iansk Sounds good. Now that we're discussing it - perhaps it would make sense to have the dev team mark these items for each endpoint as recommended or required params? (I know we can't actually edit the params in the docs right now)

Reply:
@KICKdesigns It's in the pipeline to mark params as required vs optional. In the meantime, I'd like the docs to fill the gap. You're seeing first hand what our customers need to figure out by themselves, so although this exercise is painful, it'll be worth it :)